services/harmony/test/ogc-api-coverages/describe-collections.ts

```diff
@@ -7,7 +7,7 @@ import { ServiceConfig } from '../../app/models/services/base-service';
 import { hookServices } from '../helpers/stub-service';
 
 describe('OGC API Coverages - describeCollections', function () {
-  const collection = 'C1225776654-ASF';
+  const collection = 'C1244141250-EEDTEST';
```

Comment:
The metadata for the ASF version of this Sentinel 1 collection has changed since the last time the test fixtures were run. This EEDTEST version is equivalent, and in the state the the ASF collection used to be.